### ISSUE TITLE
Target C alpha co-ordinates in m8 are not superimposed on query protein

### ISSUE THREAD

**Issue by twaks456** [steineggerlab/foldseek#66]
I made .pdb file using the data in the tca column of .m8 file from FoldSeek search server. If I open the target and query files in PyMOL, they are not superimposed, but in the search server viewer they are. Superimposition using PyMOL does not work, and it may be different to the FoldSeek superimposition anyway. Why aren't the co-ordinates in .m8 file superimposed on the query?

**Comment by martin-steinegger** [comment]
Yes, it just the normal Calpha. In our `foldseek_newscore` branch I implemented that foldseek can print out Calpha only PDBs superposed using the alignment. This will be soon in the main branch. 
@milot-mirdita we should probably also print out superposed Calpha's in the server and allow to download PDB files.

**Comment by martin-steinegger** [comment]
Calpha's are now all printable, as well as the rotation matrix and translation vector. 
We also offer a way to print out superposed PDB files using `--format-mode 5`
